tools/unix/version.sh

```diff
@@ -42,8 +42,17 @@ function android_code {
   echo "${cutYear//./}$(printf %02d "$COUNT")"
 }
 
-function git_hash {
-  git describe --match="" --always --abbrev=8 --dirty
```

Comment:
Why did you remove it? Please leave it in the string version on all platforms, it will help us to debug or to distinguish between someone's else "custom" versions.

Reply:
Git has been removed from version string on Android because it created the chicken or the egg problem (context: https://github.com/organicmaps/organicmaps/issues/825). 

- Android: `2022.08.19-7-FDroid` - generated by Gradle by using data frversion.sh
- iOS: `2022.08.19-7` - generated by Xcode, don't have a place for githash
- Qt: `2022.08.19-7 (githash)`